clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -343,9 +174,15 @@ public Struct parseResponse(short version, ByteBuffer buffer) {
                 throw new IllegalStateException("Response schema for api " + name + " for version " + i + " is null");
         }
 
+        this.messageType = messageType;
+        this.id = id;
+        this.name = name;
```

Comment:
Could we fix the comment about "name" as it is used to be a part of metrics name (see https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/network/RequestChannel.scala#L239). The origin comment is stale.

Reply:
Good catch.